Bottom: c31aa5df3e9296c2dec1f9060759dc46607015d5
Top:    943bd81fe8922d43c7b6e95c9e9d86581b5b54e8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-01 23:29:08 +0100

Refresh of hid-gtk-rest-clip-region-for-b

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 64a12a8..0b0b157 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -788,6 +788,9 @@ redraw_region (GdkRectangle *rect)
     DrawMark ();
 
   priv->clip = false;
+
+  /* Rest the clip for bg_gc, as it is used outside this function */
+  gdk_gc_set_clip_mask (priv->bg_gc, NULL);
 }
 
 void
